docs/content/configuration-and-management/tier-configuration.md

```diff
@@ -0,0 +1,239 @@
+
+```bash
+kubectl annotate llminferenceservice <model-name> -n llm \
+  alpha.maas.opendatahub.io/tiers='["stier","premium","enterprise"]' \
+  --overwrite
+```
+
+**Annotation Behavior:**
+
+- **List of tier names**: Grant access to specific tiers (e.g., `["stier","premium","enterprise"]`)
+- **Empty list `[]`**: Grant access to **all** tiers
+- **Missing annotation**: **No** tiers have access by default
+
+**Example - Grant access to stier and premium tiers:**
+
+```bash
+kubectl annotate llminferenceservice qwen3 -n llm \
+  alpha.maas.opendatahub.io/tiers='["stier","premium"]' \
+  --overwrite
+```
+
+This annotation automatically sets up the necessary RBAC (Role and RoleBinding) for the specified tiers to access the model via MaaS tokens.
+
+!!!Note "Manual RBAC Setup"
+    For reference, here's what the automatic RBAC setup looks like behind the scenes if you need to configure access manually:
+
+    ```yaml
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: Role
+    metadata:
+      name: <tier>-model-post-access
+      namespace: <model-namespace>
+    rules:
+      - apiGroups: ["serving.kserve.io"]
+        resources: ["llminferenceservices"]
+        verbs: ["post"]
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: RoleBinding
+    metadata:
+      name: <tier>-model-post-access-tier-binding
+      namespace: <model-namespace>
+    subjects:
+      - kind: Group
+        name: system:serviceaccounts:maas-default-gateway-tier-<tier>
+        apiGroup: rbac.authorization.k8s.io
+    roleRef:
+      kind: Role
+      name: <tier>-model-post-access
+      apiGroup: rbac.authorization.k8s.io
+    ```
+
+### 3. Configure Rate Limiting
+
+Add tier-specific rate limits by patching the existing `gateway-token-rate-limits` TokenRateLimitPolicy:
+
+```bash
+kubectl patch tokenratelimitpolicy gateway-token-rate-limits -n openshift-ingress --type merge --patch-file=/dev/stdin <<'EOF'
+spec:
+  limits:
+    stier-user-tokens:
+      rates:
+        - limit: 999
+          window: 1m
+      when:
+        - predicate: auth.identity.tier == "stier"
+      counters:
+        - expression: auth.identity.userid
+EOF
+```
+
+**Rate Limit Policy Configuration Explained:**
+
+1. **Target reference** - Specifies which HTTPRoute this policy applies to
+2. **Main limits section** - Container for all tier-specific rate limit definitions
+3. **Tier definition** - Each tier (free, premium, enterprise) gets its own configuration block (this is just a naming convention, it is not used for the actual tier resolution)
+4. **Request limit** - Maximum number of requests allowed per time window
+5. **Time window** - Duration after which the request counter resets
+6. **Predicate condition** - Determines when this tier's limits apply based on user authentication
+7. **Counter expression** - Tracks requests per user ID for reporting purposes
```

Comment:
This sections looks a bit detached. I wonder if it wouldn't help to bring original content [docs/content/tier-management.md](https://github.com/opendatahub-io/maas-billing/blob/07c584196c551f4ed386e9c5dd90cedcce08380a/docs/content/tier-management.md?plain=1#L142-L190) where we had sample policy first with marked sections so this numbered list was more like a footnote section explaining the concepts marked in the RateLimitPolicy resource